libs/utils/src/lsn.rs

```diff
@@ -129,6 +129,8 @@ pub struct LsnParseError;
 impl Lsn {
     /// Maximum possible value for an LSN
     pub const MAX: Lsn = Lsn(u64::MAX);
+    /// Minimum possible value for an LSN
+    pub const MIN: Lsn = Lsn(u64::MIN);
```

Comment:
nit: I don't see this being used anywhere.

Reply:
this was from an earlier version of the patch and not used anymore with the rewrite, removed